v1alpha1/meshmodel.proto

```diff
@@ -552,3 +552,71 @@ message CustomResourceKind {
 	// JSONSchema is the json schema to validate a custom resource of this kind
 	google.protobuf.Struct jsonSchema = 2;
 }
+
+// HTTPMatch defines an individual route for HTTP traffic
+message HTTPMatch {
+	// Name is the name of the HTTP match
+	string name = 1;
+
+	// Methods configures allowed HTTP method string, e.g. "GET","DELETE","POST"
+	repeated string methods = 2;
+
+	// PathRegex is a regular expression defining the route
+	string pathRegex = 3;
```

Comment:
Why not include existed `exactPath` and `pathPrefix`?

Reply:
All the names & fields are all following the SMI spec, there's no `exactPath` or `pathPrefix` in SMI.